### PR TITLE
Used consistent styling for placeholder and variable text

### LINKED ISSUE #22087: [Improvement]: Use consistent styling for placeholder and variable text
[maintainer edit]
This issue is reserved for the MLH Fellowship program.

### Is this issue for a product update or release that is not yet available to customers?

No? Then please continue. Thank you!

### What article(s) is affected?

Probably a lot, but my regex knowledge limited me from figuring out exactly how many articles. Need to find instances of `<string>` that aren't tags for list items, code, or URLs.

### Describe the desired docs changes and the problem they solve for customers.

Right now, our docs use a variety of styles for placeholder and variable text. We should be using all uppercase, but there are instances of `<strings>` and other non-uppercase styling. Similar to how we removed `<em>` tags from code examples (https://github.com/github/docs/issues/20890), this might be a good issue for OS contributors.

### Who does this affect?

People using the docs who might not recognize placeholder or variable text. Contributors trying to learn GitHub Docs style

### What is the impact to users?

Possible confusion over what text is a placeholder or variable.

### Are you willing to open a PR to make this fix? If not, do you know anyone (Hubber or not, we're open source!) who's willing to help write this documentation?

I recommend this going to our OS community :sparkle:

### PR DESCRIPTION
**Pull request by OlayinkaAtobiloye**
### Why:

Closes [ISSUE](https://github.com/github/docs/issues/22087)

### Check off the following:

- [X] I have reviewed my changes in staging (look for the "Automatically generated comment" and click the links in the "Preview" column to view your latest changes).
- [X] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### CODE REVIEW

**Comment by cmwilson21** [comment]
👋 @OlayinkaAtobiloye - Thanks for the PR! I'll get it up for review 💖

File: content/account-and-profile/managing-subscriptions-and-notifications-on-github/setting-up-notifications/configuring-notifications.md
Comment (mattpollard) [inline content/account-and-profile/managing-subscriptions-and-notifications-on-github/setting-up-notifications/configuring-notifications.md:80]: This isn't a placeholder, but it's some unnecessary emphasis that we can definitely remove. Thanks! ✨

File: content/admin/identity-and-access-management/managing-iam-for-your-enterprise/username-considerations-for-external-authentication.md
Comment (mattpollard) [inline content/admin/identity-and-access-management/managing-iam-for-your-enterprise/username-considerations-for-external-authentication.md:57]:
```suggestion
| Azure Active Directory (Azure AD) | IDP-USERNAME is formed by normalizing the characters preceding the `@` character in the UPN (User Principal Name), which does not include the `#EXT#` for guest accounts. |
```

File: content/admin/policies/enforcing-policy-with-pre-receive-hooks/creating-a-pre-receive-hook-script.md
Comment (mattpollard) [inline content/admin/policies/enforcing-policy-with-pre-receive-hooks/creating-a-pre-receive-hook-script.md:72]: Since the placeholder in the "Variable" column really is `N` (<code>$GIT_PUSH_OPTION_<em>N</em></code>), let's use "N" in the "Description" column, too.

```suggestion
| <pre>$GIT\_PUSH\_OPTION\_N</pre> | Where N is an integer starting at 0, this variable contains the push option string that was sent by the client. The first option that was sent is stored in `GIT_PUSH_OPTION_0`, the second option that was sent is stored in `GIT_PUSH_OPTION_1`, and so on. For more information about push options, see "[git-push](https://git-scm.com/docs/git-push#git-push---push-optionltoptiongt)" in the Git documentation. | abcd |{% ifversion ghes %}
```

File: content/get-started/quickstart/hello-world.md
Comment (mattpollard) [inline content/get-started/quickstart/hello-world.md:32]:
```suggestion
A repository is usually used to organize a single project. Repositories can contain folders and files, images, videos, spreadsheets, and data sets -- anything your project needs. Often, repositories include a README file, a file with information about your project. README files are written in the plain text Markdown language. You can use this [cheat sheet](https://www.markdownguide.org/cheat-sheet/) to get started with Markdown syntax. {% data variables.product.product_name %} lets you add a README file at the same time you create your new repository. {% data variables.product.product_name %} also offers other common options such as a license file, but you do not have to select any of them now.
```

File: content/pages/setting-up-a-github-pages-site-with-jekyll/adding-content-to-your-github-pages-site-using-jekyll.md
Comment (mattpollard) [inline content/pages/setting-up-a-github-pages-site-with-jekyll/adding-content-to-your-github-pages-site-using-jekyll.md:38]: _PAGE-NAME.md_ is a filename and not a placeholder, though "PAGE-NAME" is a placeholder _within_ the filename 😅 See [this](https://github.com/github/docs-internal/blob/main/contributing/content-style-guide.md#file-and-directory-names) for guidance on filenames.

```suggestion
3. In the root of your publishing source, create a new file for your page called `PAGE-NAME.md`, replacing PAGE-NAME with a meaningful filename for the page.
```

File: content/pages/setting-up-a-github-pages-site-with-jekyll/adding-content-to-your-github-pages-site-using-jekyll.md
Comment (mattpollard) [inline content/pages/setting-up-a-github-pages-site-with-jekyll/adding-content-to-your-github-pages-site-using-jekyll.md:61]: This is another [filename](https://github.com/github/docs-internal/blob/main/contributing/content-style-guide.md#file-and-directory-names) that includes a placeholder 👀

```suggestion
4. Create a new file called `YYYY-MM-DD-NAME-OF-POST.md`, replacing YYYY-MM-DD with the date of your post and NAME-OF-POST with the name of your post.
```

File: content/search-github/getting-started-with-searching-on-github/understanding-the-search-syntax.md
Comment (mattpollard) [inline content/search-github/getting-started-with-searching-on-github/understanding-the-search-syntax.md:38]: We either need to update the placeholders in the following tables to be "NUMBER", or use "N".

```suggestion
You can use the range syntax <code><em>n</em>..<em>n</em></code> to search for values within a range, where the first number N is the lowest value and the second is the highest value.
```

**Review by mattpollard — DISMISSED** [review]
Looking great, @OlayinkaAtobiloye ✨ Thanks for making these improvements!

I'll pre-approve to unblock you. I've made a few minor suggestions that address some rendering issues, and one comment could use a little feedback from @ethanpalm before you merge.

Let me know if you have any questions 🙏🏻

**Edit**: I also noticed a [check failure](https://github.com/github/docs/actions/runs/4373663171/jobs/7652120829) 👇🏻 for this PR. It looks like it's due to a merge conflict resolution. See https://github.com/github/docs/pull/24353#pullrequestreview-1332779394 👀

File: data/reusables/organizations/review-team-repository-access.md
Comment (mattpollard) [inline data/reusables/organizations/review-team-repository-access.md:3]: 💡 Turns out we removed this image [on `main`](https://github.com/github/docs/blob/main/data/reusables/organizations/review-team-repository-access.md?plain=1). We need to remove this Markdown to get the checks passing.

```suggestion
```

File: content/pages/setting-up-a-github-pages-site-with-jekyll/adding-content-to-your-github-pages-site-using-jekyll.md
Comment (sabrowning1) [inline content/pages/setting-up-a-github-pages-site-with-jekyll/adding-content-to-your-github-pages-site-using-jekyll.md:39]:
```suggestion
4. Add the following YAML frontmatter to the top of the file, replacing PAGE-TITLE with the page's title and URL-PATH with a path you want for the page's URL. For example, if the base URL of your site is `https://octocat.github.io` and your URL-PATH is `/about/contact/`, your page will be located at `https://octocat.github.io/about/contact`.
```

File: content/pages/setting-up-a-github-pages-site-with-jekyll/adding-content-to-your-github-pages-site-using-jekyll.md
Comment (sabrowning1) [inline content/pages/setting-up-a-github-pages-site-with-jekyll/adding-content-to-your-github-pages-site-using-jekyll.md:62]:
```suggestion
4. Add the following YAML frontmatter to the top of the file, replacing POST-TITLE with the post's title, YYYY-MM-DD hh:mm:ss -0000 with the date and time for the post, and CATEGORY-1 and CATEGORY-2 with as many categories you want for your post.
```

**Review by sabrowning1 — DISMISSED** [review]
This looks great! ✨

File: content/pages/setting-up-a-github-pages-site-with-jekyll/adding-content-to-your-github-pages-site-using-jekyll.md
Comment (ethanpalm) [reply content/pages/setting-up-a-github-pages-site-with-jekyll/adding-content-to-your-github-pages-site-using-jekyll.md]: I agree this is a tricky one, but I think it's fine as is. If we put just "YYY-MM-DD hh:mm:ss -0000" in quotes, then it stands out more from the others.

Maybe slightly rewrite the sentence as

```
4. Add the following YAML frontmatter to the top of the file, the post's title wrapped in quotation marks, the date and time for the post in YYYY-MM-DD hh:mm:ss -0000 format, and as many categories as you want for your post.
```

Comment (ethanpalm) [reply content/pages/setting-up-a-github-pages-site-with-jekyll/adding-content-to-your-github-pages-site-using-jekyll.md]: I can't suggest under another suggestion ☹️ so that got displayed poorly.

Comment (mattpollard) [reply content/pages/setting-up-a-github-pages-site-with-jekyll/adding-content-to-your-github-pages-site-using-jekyll.md]: Good idea. I made a couple of small tweaks. @OlayinkaAtobiloye 👋🏻 I think https://github.com/github/docs/pull/24353#discussion_r1131679811 should do it, though you'll need to edit that line in the source manually 😅

Comment (OlayinkaAtobiloye) [reply content/pages/setting-up-a-github-pages-site-with-jekyll/adding-content-to-your-github-pages-site-using-jekyll.md]: @ethanpalm thanks for the suggestion! @mattpollard, made some changes, is this good to go? Thanks!

Comment (mattpollard) [reply content/pages/setting-up-a-github-pages-site-with-jekyll/adding-content-to-your-github-pages-site-using-jekyll.md]: Yep, looks good to me 🚀

**Review by sabrowning1 — APPROVED** [review]
This looks great, thanks so much @OlayinkaAtobiloye! I've added it to the merge queue 🎉

**Pull request merged**
